composites/Plugin/SnippetEditor/tests/positionSuggestionsTest.js

```diff
@@ -61,6 +62,7 @@ describe( "positionSuggestions", () => {
 		it( "returns the horizontal position that corresponds with right", () => {
 			global.window.innerWidth = 500;
 			// ( Caret left - parent left ) - ( ( caret left + popover width + 3 ) - window.innerWidth )
+			// ( 730 - 20 ) - ( ( 730 + 220 + 3 ) - 500 )
```

Comment:
Please use multiline comments and add calculations to the two other tests as well.